docs/pages/cloud/faq.mdx

```diff
@@ -150,6 +158,10 @@ Proxy recording mode is disabled for cloud customers
 
 The ability to download recordings for offline viewing will be available in a future release.
 
+## Is there a way to forward Teleport Cloud audit events to my company's internal Security Event and Incident Managment (SEIM)?
```

Comment:
I traditionally see "SIEM" used more than "SEIM", but maybe I'm behind the times?

Reply:
Good catch. Changed in [0e596c3](https://github.com/gravitational/teleport/pull/12496/commits/0e596c3bad29e200e8a27be85236450f7babebec).